src/main/menu/actions/edit.js

```diff
@@ -22,13 +22,17 @@ ipcMain.on('mt::ask-for-image-auto-path', (e, { pathname, src, id }) => {
 })
 
 export const edit = (win, type) => {
-  win.webContents.send('AGANI::edit', { type })
+  if (win && win.webContents) {
+    win.webContents.send('AGANI::edit', { type })
+  }
 }
 
 export const screenshot = (win, type) => {
   ipcMain.emit('screen-capture', win)
 }
 
 export const lineEnding = (win, lineEnding) => {
-  win.webContents.send('AGANI::set-line-ending', { lineEnding, ignoreSaveStatus: false })
+  if (win.webContents) {
```

Comment:
We need to check window first (`win && win.webContents`)?

Reply:
Yes, my fault.